quickwit/quickwit-telemetry/src/payload.rs

```diff
@@ -62,20 +62,22 @@ impl From<TelemetryEvent> for EventWithTimestamp {
 }
 
 /// Represents a Telemetry Event send to Quickwit's server for usage information.
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq)]
 pub enum TelemetryEvent {
-    /// Create command is called.
-    Create,
-    /// Ingest command is called.
-    Ingest,
-    /// Delete command
-    Delete,
-    /// Garbage Collect command
+    /// Create index API endpoint is called.
+    CreateIndex,
+    /// Delete index API endpoint is called.
+    DeleteIndex,
+    /// Garbage collect command is called.
     GarbageCollect,
-    /// Serve command is called.
+    /// Run service command is called.
     RunService(String),
     /// EndCommand (with the return code)
     EndCommand { return_code: i32 },
+    /// Sent every once in a while (used for deducing uptime).
```

Comment:
```suggestion
    /// Event sent every 12h to signal the server is running.
```

Reply:
I tend to avoid being this explicit in comments 😄 . someone needs to remember to update the `12h` if the interval ever gets updated.